pkg/cloud/gcp/gcs_kms.go

```diff
@@ -24,7 +24,10 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
-const gcsScheme = "gs"
+const (
```

Comment:
should we rename this file to `gcp_kms.go`?

Reply:
done!